src/CommunityToolkit.Maui.Core/Interfaces/IAvatarView.shared.cs

```diff
@@ -9,6 +9,43 @@ public interface IAvatarView : IBorderView, ILabel, Microsoft.Maui.IImage, IImag
 	/// <summary>Gets a value indicating the avatar border width.</summary>
 	double BorderWidth { get; }
 
+	/// <summary>Gets or sets a value of the control image cache validity.</summary>
+	TimeSpan CacheValidity { get; }
+
+	/// <summary>Gets or sets a value indicating whether the control image cache is enabled.</summary>
+	bool CachingEnabled { get; }
+
 	/// <summary>Gets a value indicating the avatar corner radius <see cref="Microsoft.Maui.CornerRadius"/>.</summary>
 	CornerRadius CornerRadius { get; }
+
+	/// <summary>Gets or sets a value of the control default gravatar property.</summary>
+	DefaultGravatarImage DefaultGravatar { get; }
```

Comment:
Can we change this to be `Image`, then devs can use this not only for GravatarImage. Of course we can create a behavior or attached properties to abstract the Gravatar implementation, but I would like to have this more _global_

Reply:
@pictos  I'm personally not sure we should do this.  It could cause a lot of network traffic, which may not be appropriate for all consumers.

Instead, if we want to provide default Images, we *could* provide a selection from embeded resources, but this would increase the size of the package for all users, if they use AvatarView or not.

The consumer COULD if they want, provide a bad email address and set the default gravatar image, thus forcing it to display an Image.  This would then is the consumers choice to implement.  We are not stopping them from doing so, should they wish.

Reply:
Well, how this can be different from using the Image from the web in terms of `network traffic` ? I can see a lot of users that would like to use the avatar view with other web resources, like apps that have a profile, etc.